templates/boot_loader_conf/bootcfg_esxi70u1.template

```diff
@@ -0,0 +1,8 @@
+bootstate=0
+title=Loading ESXi installer
+prefix=/images/esxi7-x86_64
+kernel=b.b00
+kernelopt=runweasel ks=http://10.8.10.121:80/cblr/svc/op/ks/profile/esxi7-x86_64
```

Comment:
As per the already existing comment from @mpsOxygen this needs to be changed.

Reply:
Fixed in the last commit.  Thanks, @SchoolGuy & @mpsOxygen.